clang/lib/CConv/CheckedRegions.cpp

```diff
@@ -155,10 +164,8 @@ bool CheckedRegionFinder::isInStatementPosition(CallExpr *C) {
   auto Parent = Parents[0].get<CompoundStmt>();
   if (Parent) {
     //Check if we are the only child
-    int NumChilds = 0;
-    for (const auto &child : Parent->children()) {
-      NumChilds++;
-    }
+    auto childs = Parent->children();
+    int NumChilds = std::distance(childs.begin(), childs.end());
```

Comment:
Is this computing the same thing as the old code, but perhaps more efficiently? I'm curious what it's actually doing. E.g., if we want the size of the `Parent->children()` sequence, presumably there's a direct method that can give you that?

Reply:
Computing the size of the sequence. AFAIK there is not a better way of doing it.

Reply:
No `::size()` method (or similar) ?

Reply:
Not as far as I can see. Here's the type: [llvm::iterator_range](https://llvm.org/doxygen/classllvm_1_1iterator__range.html).